Content.Shared/Hands/EntitySystems/SharedHandsSystem.cs

```diff
@@ -453,4 +463,23 @@ public int CountFreeableHands(Entity<HandsComponent> hands)
 
         return freeable;
     }
+
+    /// <summary>
+    /// Counts the number of hands that can are empty or can be emptied by dropping an item, ignoring the hand the <paramref name="except"/> entity is in.
+    /// Unremoveable items will cause a hand to not be freeable.
+    /// </summary>
+    public int CountFreeableHandsExcept(Entity<HandsComponent> hands, EntityUid except)
```

Comment:
Wouldn't it be more convenient (for the coder calling this function) to just fold this into the original function by making the `except` parameter optional and default to null?